src/battle.ts

```diff
@@ -2077,6 +2077,9 @@ class Battle {
 					poke.itemEffect = 'bestowed';
 					this.scene.resultAnim(poke, item.name, 'neutral');
 					break;
+				case 'poltergeist':
+					poke.itemEffect = 'poltergeisted';
```

Comment:
(Going by the German origin of the word, the Geist here is the Ghost-type Pokémon, and the Polter actually refers to the sounds of the objects (typically chains) being moved. Maybe "disturbed" would be a better choice?)

Reply:
I went with disturbed. thanks!